biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -223,6 +223,12 @@
 
 	String				RESOLVE										= "-resolve";
 
+	/**
+	 * Exclude the system resource from the resulting wiring in resolve. The
+	 * default is true
+	 */
+	String				RESOLVE_EXCLUDESYSTEM						= "-resolve.excludesystem";
```

Comment:
I think this needs to be called `-resolve-excludesystem`. Otherwise it looks like a merge property for `-resolve`.

Reply:
Resolve is not a merge property,it can only be a single value.

Reply:
Today. But this will preclude it ever being a merge property. And it also looks like a merge property.  No other instruction uses a `.` to separate words. They all use `-`. e.g. `-dsannotations-options`. I think using a `.` here is a bad precedent when we can easily use `-`.